aspnetcore/fundamentals/logging/loggermessage.md

```diff
@@ -17,45 +16,63 @@ By [Luke Latham](https://github.com/guardrex)
+## Define and DefineScope
+
+The string provided to the `Define` and `DefineScope` methods is a template and not an interpolated string. Placeholders are filled in the order that the types are specified. Placeholder names in the template should be descriptive and consistent across templates.  Placeholder names should be descriptive and consistent because they serve as property names within structured log data. We recommend [Pascal casing](/dotnet/standard/design-guidelines/capitalization-conventions) for placeholder names. For example, `"{Count}", "{FirstName}"`.
 
 ## Implementing LoggerMessage.Define
 
 Each logging message is an `Action` held in a static field created by `LoggerMessage.Define`. For example, the sample app creates a field to describe a log message for a GET request for the Index page (*Internal/LoggerExtensions.cs*):
 
 [!code-csharp[Main](loggermessage/sample/Internal/LoggerExtensions.cs?name=snippet1)]
 
-For the `Action`, specify a log level, a unique event identifier (`EventId`) with the name of the static extension method, and the message template. For a request for the Index page of the sample app, the log level is `Information`, the event id is `1` with the name of the `IndexPageRequested` method, and the template is a string:
+For the `Action`, specify:
+
+* The log level.
+* A unique event identifier (`EventId`) with the name of the static extension method.
+* The message template. 
+
+A a request for the Index page of the sample app:
+
+* The log level is `Information`.
+* The event id is `1` with the name of the `IndexPageRequested` method.
+* The template is a string.
 
 [!code-csharp[Main](loggermessage/sample/Internal/LoggerExtensions.cs?name=snippet5)]
 
-When you supply the name of the method with your event id, it's used by structured logging stores to provide richer event logging than supplying a numbered event id alone.
+When the name of the method is supplied with the event id:
+
+* The method name is used by structured logging stores. 
+* The method name provides richer event logging than the numbered event id alone.
+
+The `Action` is invoked through a strongly-typed extension method. 
 
-The `Action` is invoked through a strongly-typed extension method. In the sample app, the method that logs a message for an Index page GET request is `IndexPageRequested`:
+The `IndexPageRequested` logs a logs a message for an Index page GET request in the sample app:
 
 [!code-csharp[Main](loggermessage/sample/Internal/LoggerExtensions.cs?name=snippet9)]
 
-In the Index page's code-behind file, `IndexPageRequested` is called on the logger to log the message (*Pages/Index.cshtml.cs*):
+`IndexPageRequested` logs the message in *Pages/Index.cshtml.cs*:
 
 [!code-csharp[Main](loggermessage/sample/Pages/Index.cshtml.cs?name=snippet2&highlight=3)]
 
-You can inspect the message in the app's console output:
+The message can be inspected in the app's console output:
```

Comment:
You seem to be removing "you," but AFAIK the style manual hasn't changed on this point. Let me know if there's been a new ruling on it. I also hate "be/being" verbs. Come to think of it, it would probably be best to just take "you" out and make them active instructions:

> Inspect in the app's console output: